Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	
+++ b/main.py	
@@ -34,9 +34,9 @@
 text_frame.place(x=0, y=0)
 
 
-text_frame_text = Label(root, text="Rock Paper Scissors Game", bg="#79aed9", fg="black", font=("consolas", 20, "bold"))
+text_frame_text = Label(root, text="Rock Paper Scissors Game", bg="#79aed9", fg="black", font=("consolas", 22, "bold"))
 text_frame_text.place(x=200, y=10)
-
+#a
 
 player_avatar = Label(root, image=player_image, borderwidth=0, bg="#3f6a99")
 player_avatar.place(x=34, y=110)
